composer.json

```diff
@@ -23,26 +23,38 @@
         }
     ],
     "require": {
-        "php": "^5.5 || ^7.0",
-        "zendframework/zend-eventmanager": "^2.6 || ^3.0",
-        "zendframework/zend-http": "^2.5.2",
-        "zendframework/zend-mvc": "^2.5.2",
-        "zendframework/zend-servicemanager": "^2.7.5 || ^3.0.3"
+        "php": "^5.6 || ^7.0",
+        "zendframework/zend-eventmanager": "^2.6.3 || ^3.0.1",
+        "zendframework/zend-http": "^2.5.5",
+        "zendframework/zend-mvc": "^2.7.9 || ^3.0.2",
+        "zendframework/zend-servicemanager": "^2.7.6 || ^3.1.1"
     },
     "require-dev": {
-        "zendframework/zend-modulemanager": "^2.6.1",
+        "zendframework/zend-modulemanager": "^2.7.2",
         "zendframework/zend-config": "^2.6",
-        "zendframework/zend-view": "^2.5.2",
-        "zendframework/zend-serializer": "^2.6",
-        "zendframework/zend-log": "^2.5.2",
-        "zendframework/zend-i18n": "^2.6",
-        "phpunit/phpunit": "^4.8",
-        "squizlabs/php_codesniffer": "1.4.*",
-        "satooshi/php-coveralls": "~0.6"
+        "zendframework/zend-view": "^2.8.1",
+        "zendframework/zend-serializer": "^2.8",
+        "zendframework/zend-log": "^2.9.1",
+        "zendframework/zend-i18n": "^2.7.3",
```

Comment:
Some cleanup should be done here. I remember that I've added all of this crap in the past due to all those dependencies being needed, but they are actually not used.